im.srain.Srain.json

```diff
@@ -0,0 +1,146 @@
+{
+  "app-id": "im.srain.Srain",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "srain",
+  "rename-desktop-file": "Srain.desktop",
+  "rename-appdata-file": "Srain.appdata.xml",
+  "rename-icon": "srain",
+  "copy-icon": true,
+  "finish-args": [
+    "--share=ipc",
+    "--device=all",
+    "--filesystem=home:ro",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=session-bus",
```

Comment:
What services does it need to talk to? You need to specifically enumerate them rather than asking for everything.